grafana/overlays/nerc-ocp-obs/patches/grafana-route.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  host: grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu
```

Comment:
We'll want to change `infra` to `obs` here I think.

Reply:
thanks, yes, i created this draft to work with Jeet on it together.

Reply:
```suggestion
  host: grafana.apps.nerc-ocp-obs.rc.fas.harvard.edu
```